### PR TITLE
Updating sample data

### PR DESCRIPTION
**Pull request by martinkunc**
Updates generated documentation and bringing new samples to docs/insights-archive-sample.
There are no code changes here.

### CODE REVIEW

File: docs/gathered-data.md [hunk 1]
```diff
@@ -180,9 +181,9 @@ Output raw size: 108
 #### MostRecentMetrics
 [{"Name":"config/metrics","Captured":"0001-01-01T00:00:00Z","Fingerprint":"","Item":"SGVsbG8sIGNsaWVudAo="}]
 
-## UnhealthyNodes
+## Nodes
 
-collects all unhealthy Nodes.
+collects all Nodes.
```
Comment (iNecas) [inline docs/gathered-data.md:186]: There is still the `unhealthy` word referenced in the the next paragraph, that seems to not be relevant anymore

**Comment by iNecas** [comment]
Small comment about the content itself.

**Comment by martinkunc** [comment]
@iNecas  Thanks for the comments :)

**Pull request merged**